package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -70,6 +70,19 @@ const parse: ParseFunction = (capture, parse, state) => ({
   content: parseInline(parse, capture[0], state),
 });
 
+const getUrlsFromText = (text: string) => {
+  try {
+    return anchorme(text, {
+      list: true,
+    });
+  } catch (e) {
+    console.log(`failure at getUrlsFromText: ${e}`);
```

Comment:
Maybe it's worth using `console.warn` for this, to make it stand out as a warning in the console? Maybe it could help us find edge cases we wouldn't think of while working with the SDK? Not sure, just an idea